src/nni_manager/training_service/reusable/trialDispatcher.ts

```diff
@@ -578,6 +578,15 @@ class TrialDispatcher implements TrainingService {
 
         }
     }
+    
+    private async initEnvironments(): Promise<void> {
```

Comment:
prefetchEnvironments may be more accurate.

Reply:
updated.